android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -560,15 +847,7 @@ public void mediaStreamRelease(String id) {
                 Log.d(TAG, "mediaStreamRelease() stream is null");
                 return;
             }
-
             localStreams.remove(id);
-
-            // MediaStream.dispose() may be called without an exception only if
-            // it's no longer added to any PeerConnection.
-            for (int i = 0, size = mPeerConnectionObservers.size(); i < size; i++) {
-                mPeerConnectionObservers.valueAt(i).removeStream(stream);
```

Comment:
We might need to removeTrack instead here, in order to accomplish the same. Even though I no longer see why MediaStream.dispose() would fail...

Reply:
Internally, `MediaStream.dispose()` calls `removeTrack`. I do not see either why we cannot use `MediaStream.dispose()` here. It currently is being used.

Reply:
👍